build/git.go

```diff
@@ -36,3 +36,28 @@ func gitHasChanges(path string) (bool, error) {
 	data = bytes.TrimSpace(data)
 	return len(data) != 0, nil
 }
+
+func gitInit(path string) error {
+	if out, err := exec.Command("git", "init", path).CombinedOutput(); err != nil {
+		return fmt.Errorf("cannot init a repository: %v\n%s\n", err, string(out))
+	}
+	return nil
+}
+
+func gitAdd(root, file string) error {
```

Comment:
Maybe we could make this

```go
func gitAdd(root string, files ...string) error {
   git := exec.Command("git", append([]string{"add"}, files...)...)
   git.Dir = root
   …
   return nil
}
```

Then we wouldn't need a loop: `gitAdd(root, "Dockerfile", "go.mod", "go.sum")`

Reply:
Considered this. Will do.